frontend/src/screens/dashboard/docsqa/index.tsx

```diff
@@ -61,7 +61,7 @@ const DocsQA = () => {
   const [modelConfig, setModelConfig] = useState(defaultModelConfig)
   const [retrieverConfig, setRetrieverConfig] = useState(defaultRetrieverConfig)
   const [promptTemplate, setPromptTemplate] = useState(defaultPrompt)
-  const [isStreamEnabled, setIsStreamEnabled] = useState(true)
+  // const [isStreamEnabled, setIsStreamEnabled] = useState(true)
```

Comment:
are we removing streaming toggle option? if yes, then remove this state completely